packages/grafana-ui/src/components/uPlot/MicroPlot.tsx

```diff
@@ -0,0 +1,376 @@
+import React, { PureComponent } from 'react';
```

Comment:
Ignore this file, look at UPlotChart.tsx instead. This is POC that @ryantxu did some time ago. Not sure why it shows here even though this PR is targeting the source branch